### PR TITLE
Python 3 - pathod.log

### PR DESCRIPTION
**Pull request by dufferzafar**

### CODE REVIEW

File: test/pathod/test_log.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import StringIO
 from pathod import log
 from netlib.exceptions import TcpDisconnect
-import netlib.tcp
 
+from six.moves import cStringIO
```
Comment (mhils) [inline test/pathod/test_log.py]: I think we can just use `io.BytesIO` here.

Comment (dufferzafar) [inline test/pathod/test_log.py]: Then `pathod.log` will have to be modified to work with `bytes`

**Comment by mhils** [comment]
Uh, I think my last comment was moot. We probably want to log unicode?

**Comment by dufferzafar** [comment]
:confused:

**Comment by mhils** [comment]
I merged cdc4acc689673141a928e2ee3e5e2b753a28a8f7 into master and fixed the tests. We're set here! :smiley:

**Pull request closed without merging**